Framework/Source/Utils/DXHeader.h

```diff
@@ -0,0 +1,167 @@
+/***************************************************************************
```

Comment:
This sucks. It also can't be under NVIDIA license.
We need a matching CPP file that makes sure that the macros here match the DX macros (a lot of static_asserts in DX build)